recipe/meta.yaml

```
@@ -142,6 +146,10 @@ outputs:
         - cuda-version {{ cuda_version }}
       run:
         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+         # In 12.0/12.1 libcublas runtime will try to dlopen nvrtc due to a bug
+         # so it must be listed as a runtime dependency.
+         # See https://github.com/conda-forge/libcublas-feedstock/issues/9
+         - cuda-nvrtc
```

Should fix the linter error

```suggestion
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        # In 12.0/12.1 libcublas runtime will try to dlopen nvrtc due to a bug
        # so it must be listed as a runtime dependency.
        # See https://github.com/conda-forge/libcublas-feedstock/issues/9
        - cuda-nvrtc
```

---

Done in 1c15b6f.

---

Nvm looks like this was added while writing this 😅